http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -390,7 +395,11 @@ public enum AuthOutcome {
     }
 
     public static String generateId() {
-        return UUID.randomUUID().toString();
```

Comment:
The `generateId` method is also used for generating a unique identifier for the JWT ID claim and for the state code. I think we should leave this existing method unchanged and introduce a new method for the nonce generation.

Reply:
done